composer.json

```
@@ -17,7 +17,7 @@
         "knplabs/knp-time-bundle": "1.9.0",
         "lexik/maintenance-bundle": "2.1.4",
         "symfony/finder": "3.4.7 || 4.0.7",
-        "symfony/framework-bundle": "4.2.7",
+        "symfony/framework-bundle": "4.2.7 || > 5.2.5",
```

```suggestion
        "symfony/framework-bundle": "4.2.7 || 5.2.5",
```

AFAIR, we never use unbound version constraints for conflicts. If the issue is not fixed in version 5.2.6, we would update this file again.

---

I think we do?
https://github.com/contao/conflicts/commit/8df62d28c5e2dd67380d1503e490285d2ef24362
https://github.com/contao/conflicts/commit/8166d09b3b0ea81f71e4c0af7b533eac5104902a

---

Unbound version constraints bear the risk of being forgotten and suddenly people do not get updates of the package anymore without even noticing. I though that was the reasoning behind not using unbound version constraints.

But let's wait what @ausi says.

---

We'll I'm following https://github.com/symfony/symfony/issues/40618 and we will certainly change this as soon as another fix is available 🤷

---

I’m OK with the open version constraint. We should IMO create an issue to change that, and mark that issue as blocked by symfony/symfony#40618